### PR TITLE
date time picker initial time fix

### PR DESCRIPTION
**Pull request by vasilich6107**
Fix for #168

### CODE REVIEW

**Comment by vasilich6107** [comment]
@GiancarloCode any thoughts/comments?

File: packages/flutter_form_bloc/lib/src/date_time/date_time_field_bloc_builder_base.dart [hunk 1]
```diff
@@ -235,16 +235,22 @@ class _DateTimeFieldBlocBuilderBaseState<T>
   }
 
   Future<TimeOfDay> _showTimePicker(BuildContext context) async {
```
Comment (Chralu) [inline packages/flutter_form_bloc/lib/src/date_time/date_time_field_bloc_builder_base.dart:237]: To reduce conditions imbrication, you could early exit style :

```dart
class _DateTimeFieldBlocBuilderBaseState<T> ... {

  TimeOfDay _initialTime() {
    if (widget.dateTimeFieldBloc.state.value == null) {
      return widget.initialTime;
    }
    if (widget.type == DateTimeFieldBlocBuilderBaseType.time) {
      return widget.dateTimeFieldBloc.state.value;
     }
     return TimeOfDay.fromDateTime(widget.dateTimeFieldBloc.state.value);
  }

  Future<TimeOfDay> _showTimePicker(BuildContext context) async {
    return await showTimePicker(
      context: context,
      useRootNavigator: widget.useRootNavigator,
      initialTime: _initialTime(),
      builder: widget.builder,
    );
  }
}
```

Comment (vasilich6107) [reply packages/flutter_form_bloc/lib/src/date_time/date_time_field_bloc_builder_base.dart:237]: It seems that no one cares about this project anymore(

**Comment by WahdanZ** [comment]
added on version  0.20.2

**Pull request closed without merging**